Add disclaimer about OAuth credentials in source

lyzadanger opened the pull request:
I had a bit of a panic in the Tessel hands-on workshop at [Node Interactive](http://events.linuxfoundation.org/events/node-interactive) because I thought the inclusion of the OAuth credentials for the `@TesselTweet` account in source might have been inadvertent.

This bitsy PR adds additional disclaimer text to make it more clear that this was intentional. :) Sorry for being all alarmist!

lyzadanger committed:
Add disclaimer about OAuth credentials in source

johnnyman727 commented:
Great branch name :)

We'll need you to sign [the Dojo Foundation CLA](http://dojofoundation.org/about/claForm) before we can merge.

@Frijol could you review?

lyzadanger commented:
@johnnyman727 Signed, cheers.

johnnyman727 commented:
Fantastic, thank you! :clap:

Frijol commented:
@lyzadanger thanks for the PR, and for beta testing my workshop!

I'm a bit worried about how much block text this puts on the page – I've found that the more words we have, the fewer people read.

![screen shot 2015-12-09 at 8 47 31 pm](https://cloud.githubusercontent.com/assets/454690/11707235/64d825f2-9eb6-11e5-8565-9481548e4c8d.png)

Would it help to just add another comment line in the code itself?

Frijol commented on tweet.md:
```diff
@@ -7,15 +7,15 @@
 
-This code snippet makes Tessel send a tweet from a dummy account <a href="https://twitter.com/TesselTweet" target="0">(@TesselTweet)</a> that we've created for this purpose.
-
+You can use this code snippet to sent a tweet from the dummy Twitter account <a href="https://twitter.com/TesselTweet" target="0">(@TesselTweet)</a>.
```
whoops, typo - "sent" > "send"

Frijol commented on tweet.md:
```diff
@@ -7,15 +7,15 @@
 
 Because Tessel runs Node, we can leverage the [thousands of libraries](//npmjs.com) that the Node community has made.
 
-This code snippet makes Tessel send a tweet from a dummy account <a href="https://twitter.com/TesselTweet" target="0">(@TesselTweet)</a> that we've created for this purpose.
```
Why cut the "that we've created for this purpose"?

Frijol commented on tweet.md:
```diff
@@ -48,6 +48,8 @@ var twitterHandle = '@tesselproject';
 var status = 'Hello ' + twitterHandle + '. This is your #Tessel 2 speaking.';
 
 // Enter the oauth key and secret information
```
Don't need this and the new comments, cut this line

Frijol commented on tweet.md:
```diff
@@ -48,6 +48,8 @@ var twitterHandle = '@tesselproject';
 var status = 'Hello ' + twitterHandle + '. This is your #Tessel 2 speaking.';
 
 // Enter the oauth key and secret information
+// These OAuth credentials are for the dummy @TesselTweet account
```
This & the next line are good, maybe that's all it needs? (As in, can we cut the new paragraph above?)

lyzadanger committed:
Reduce wordiness, restrict to code per feedback

Addressing feedback including
https://github.com/tessel/t2-start/pull/76/files#r47189237

lyzadanger commented:
@Frijol Thanks. Feedback addressed in latest (https://github.com/lyzadanger/t2-start/commit/f63ac90151cda8e75e18d57db76f53a350b46984). Look OK?

Frijol commented:
Looks great, thanks! Merging.
